src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -226,7 +226,9 @@ def update_stats(self, area, simulation_status, progress_info):
 
         self.cumulative_bills.update_cumulative_bills(area)
 
-        self.market_unmatched_loads.update_unmatched_loads(area)
+        self.market_unmatched_loads.update_unmatched_loads(
+            self.area_result_dict, self.flattened_area_core_stats_dict, self.current_market
```

Comment:
Here i like to mention, what i wanted to mention already in 2703: 
Lets stick to the naming convention we implicitly have and rename this to `self. current_market_time_slot_str`, because i would expect a market object here judging from the current naming.
so, i would rename like this:
        self.current_market --> self.current_market_time_slot_str
        self.current_market_unix --> self.current_market_time_slot_unix
        self.current_market_datetime --> self.current_market_time_slot

Would you also like this?

Reply:
Okay, doing it now, just to get your memory recalled, you were skeptical to be so verbose back then (https://github.com/gridsingularity/d3a/pull/862#discussion_r484862864) but thats minor and will adapt this comment and be verbose

Reply:
@faizan2590 OMG, yes. Thanks for reminding me. But see above, i mean the suffix of the name. to be consistent with the market slot naming inside of the d3a code. If you like, please remove the "current" then. so :
self.current_market --> self.market_time_slot_str
self.current_market_unix --> self.market_time_slot_unix
self.current_market_datetime --> self.market_time_slot